frame/crowdloan-rewards/Cargo.toml

```diff
@@ -17,32 +17,45 @@ version = "2.0.0"
 
 [dev-dependencies]
 serde = "1.0.119"
-balances = { package = "pallet-balances", git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.13", default-features = false }
+# orml-traits = { git = "https://github.com/open-web3-stack/open-runtime-module-library", rev = "17a791edf431d7d7aee1ea3dfaeeb7bc21944301", default-features = false }
 sp-keystore = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.13" }
+libsecp256k1 = { version = "0.7.0", default-features = false, features = [
+    "hmac",
+    "static-context",
+] }
 hex-literal = "0.3"
-libsecp256k1 = { version = "0.7.0", default-features = false, features = [ "hmac", "static-context"] }
+balances = { package = "pallet-balances", git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.13", default-features = false, features = [
+    "std",
+] }
 
 [dependencies]
+# should be a dev-dependency. see: https://github.com/rust-lang/cargo/issues/6915
+# balances = { package = "pallet-balances", git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.13", default-features = false }
+
```

Comment:
Can we remove that?

Reply:
yep, I will go through and remove comments. Forgot to do that